direnvrc

```diff
@@ -1,5 +1,122 @@
 # shellcheck shell=bash
 
+_nix_import_env() {
+  local env=$1
+
+  local term_backup=$TERM path_backup=$PATH
+  if [[ -n ${TMPDIR+x} ]]; then
```

Comment:
The `+x` trick is not needed in bash anymore, you can use `if [[ -z $TMPDIR ]]; then`. In POSIX sh you would also have to escape the `"$TMPDIR"`, which is also not needed when using the `[[ ]]` test syntax (but only on the RHS).